packages/blade/src/components/Amount/Amount.tsx

```diff
@@ -19,6 +19,43 @@ import { Text } from '~components/Typography';
 import { opacity } from '~tokens/global';
 import type { FontFamily } from '~tokens/global';
 
+/**
+ * Pollyfill function to get around the node 18 error
+ *
+ * This function is maintained by i18nify team. Reach out to them for any change regarding this.
```

Comment:
Should this come from i18nify library itself then instead of being in blade repo?

Reply:
I did suggest them this but they are saying they'll just document it on their end in documentation and consumers will have to add it. Can discuss it later separately as well. Wanted to get it resolved earlier for negative amount thing. But yup overall I too feel this should be in their repo